core/payment/src/dao/activity.rs

```diff
@@ -80,7 +79,6 @@ pub fn increase_amount_paid(
     amount: &BigDecimalField,
     conn: &ConnType,
 ) -> DbResult<()> {
-    assert!(amount > &BigDecimal::zero().into()); // TODO: Remove when payment service is production-ready.
```

Comment:
Called by:
- requestor => service/notify_payment
- provider => service/send_payment

There are no checks if activity payments are 0, should they be added? if yes, requestor, provider or both

Reply:
Provider should check in `verify_payment` if all activity payment and agreement payments have amount > 0.

Reply:
added this check :)